Dockerfile

```diff
@@ -99,9 +99,16 @@ RUN curl https://raw.githubusercontent.com/aws/aws-sdk-net/master/sdk/src/Core/e
 RUN curl https://raw.githubusercontent.com/cycloidio/cycloid-cli/master/scripts/cy-wrapper.sh > /usr/bin/cy \
     && chmod +x /usr/bin/cy
 
+# Install Changie
+RUN wget -O changie.tar.gz https://github.com/miniscruff/changie/releases/download/v1.7.0/changie_1.7.0_linux_amd64.tar.gz \
+    && tar xf changie.tar.gz changie \
+    && chmod +x changie \
+    && mv changie /usr/bin/changie
+
 # Contain ec2.py dynamic inventory from https://github.com/ansible-collections/community.aws/tree/main/scripts/inventory or https://github.com/ansible/ansible/blob/stable-2.9/contrib/inventory/ec2.py
 COPY files/ansible /etc/ansible/
 COPY scripts/* /usr/bin/
 
-# Install gcp_compute inventory plugin
-RUN ansible-galaxy collection install google.cloud
+# Install Ansible galaxy collections
+RUN ansible-galaxy collection install google.cloud \
```

Comment:
I was wondering about it, if it could be easier later on to troubleshoot in 2 separated command or if I just should put both in the same

Reply:
galaxy output logs look split by roles so I guess it could be good for troubleshooting

```
Starting collection install process
Installing 'google.cloud:1.0.2' to '/root/.ansible/collections/ansible_collections/google/cloud'
Installing 'ansible.windows:1.10.0' to '/root/.ansible/collections/ansible_collections/ansible/windows'
```

Reply:
done